addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -238,13 +238,17 @@ export class AddPageTemplatePreview extends Component {
         for (const previewEl of wrapEl.querySelectorAll(".o_new_page_snippet_preview")) {
             previewEl.remove();
         }
-        this.env.addPage(wrapEl.innerHTML);
+        this.env.addPage(wrapEl.innerHTML, this.props.template.name && _t("Copy of %s", this.props.template.name));
     }
 }
 
 export class AddPageTemplatePreviews extends Component {
     static template = "website.AddPageTemplatePreviews";
     static props = {
+        isCustom: {
+            type: Function,
+            optional: true,
+        },
```

Comment:
Isn't it a boolean ? :thinking: since you set it to `True` above in `addons/website/controllers/main.py`